cmd/upgrade/upgrade.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+namespace=kubesphere-logging-system
+deployment="fluentbit-operator"
```

Comment:
```suggestion
fluentbit-operator="fluentbit-operator"
```

Reply:
It has been modified, it is better not to have `-` in the shell name, I modified it to `FluentbitOperator`